src/components/FamilyDetails.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="bg-background d-flex">
+    <v-sheet class="Family-Details">
+      <img src="../assets/images/splashFrame.png" alt="Frame" />
+      <v-card class="px-6">
+        <v-card-title class="SignupHeading mb-5">
+          <router-link to="/PersonalInfo">
+            <img src="../assets/icons/back.png" alt="Logo" class="Backicon" />
+          </router-link>
+          <p class="mx-auto">Add Your Family Details</p>
+        </v-card-title>
+        <v-divider class="mx-auto" width="361"></v-divider>
+        <v-form>
+          <v-row class="d-block py-6">
+            <v-col class="SignupInputfield pt-0">
+              <label for="" class="Baseinputlabel">First Name</label>
+              <div class="d-flex justify-space-between">
+                <BaseInput class="mr-1" placeholder="First Name" />
+                <BaseInput class="ml-1" placeholder="Last Name" />
+              </div>
+            </v-col>
+
+            <v-col class="SignupInputfield">
+              <label for="" class="Baseinputlabel">Date of Birth</label>
+              <div class="d-flex justify-space-between">
+                <BaseInput class="mr-1" placeholder="Day" />
+                <BaseInput class="" placeholder="Month" />
+                <BaseInput class="ml-1" placeholder="Year" />
+              </div>
+            </v-col>
+            <v-col class="SignupInputfield pt-0">
+              <label for="" class="Baseinputlabel">Dieases</label>
+              <div class="MultipleSelect">
+                <v-select
```

Comment:
Create a base select component and use that. This way you won't have to repeat the styles associated with it. Consider overriding the default vuetify component scss variabels. Read the documentation about this